### ISSUE TITLE
[5.6.0] ExecutionFailed: None.get

### ISSUE THREAD

**Issue by CelineWuest** [neo4j/neo4j#13105]
When running the following query:
```cypher
MERGE (x:A) RETURN EXISTS { () } ORDER BY x
```

I encountered the following error:
```
Neo4jError: Neo.DatabaseError.Statement.ExecutionFailed (None.get)
```

I believe the query mentioned above is semantically and syntactically correct and thus no error should be thrown here.

I encountered this issue when testing queries against the **Neo4j 5.6.0 community version** in a Docker container running **alpine v.3**.
On Neo4j 5.5.0, this query runs successfully and no exception is raised.

### Steps to reproduce
Run the following query and observe it throws an error:
```cypher
MERGE (x:A) RETURN EXISTS { () } ORDER BY x
```

### Expected behavior
The query should run successfully

### Actual behavior
Neo4j 5.5.0: The query runs successfully and no exception is raised
Neo4j 5.6.0: The query fails with the error message `Neo4jError: Neo.DatabaseError.Statement.ExecutionFailed (None.get)`

**Comment by Lojjs** [comment]
@DominicWuest Thanks for reporting, this sounds like a bug and we will take a look. The `ORDER BY x` should be supported, but shouldn't affect the result of the query. Therefore, as a workaround, you can omit from the query.

Best regards Louise, Neo4j Cypher team

**Comment by gem-neo4j** [comment]
Hi, this was fixed and released with Neo4j 5.9 :)
